doc/tutorials/running_deb_as_user_vm.rst

```diff
@@ -0,0 +1,245 @@
+
+    d. Select **Forward** if you do not need to make customized CPU settings.
+
+    e. Choose **Create a disk image for virtual machine**. Set the storage to 20 GB or more if necessary and click **Forward**.
+
+    f. Rename the image if you desire. You must check the **customize configuration before install** option before you finish all stages.
+
+#. Verify that you can see the Overview screen as set up, as shown in :numref:`debian10-setup` below:
+
+    .. figure:: images/debian-uservm-3.png
+       :align: center
+       :name: debian10-setup
+
+       Debian Setup Overview
+
+#. Complete the Debian installation. Verify that you have set up a vda disk partition, as shown in :numref:`partition-vda` below:
+
+    .. figure:: images/debian-uservm-4.png
+       :align: center
+       :name: partition-vda
+
+       Virtual Disk (vda) partition
+
+#. Upon installation completion, the KVM image is created in the ``/var/lib/libvirt/images`` folder. Convert the `gcow2` format to `img` **as the root user**:
+
+   .. code-block:: none
+
+      $ cd ~/debian10
+      $ qemu-img convert -f qcow2 -O raw /var/lib/libvirt/images/debian10.qcow2 debian10.img
+
+Launch the Debian Image as the User VM
+**************************************
+
+Re-use and modify the `launch_win.sh` script in order to launch the new Debian 10 User VM.
+
+.. note:: This tutorial assumes SATA is the default boot drive; replace "/dev/sda1" mentioned below with "/dev/nvme0n1p1" if you are using an NVMe drive.
+
+1. Copy the debian.img to your NUC:
+
+   .. code-block:: none
+
+      # scp ~/debian10/debian10.img user_name@ip_address:~/debian10.img
+
+#. Log in to the ACRN Service VM; generate the launch script and copy OVMF to the image directory:
+
+   .. code-block:: none
+
+      $ cd ~
+      $ cp /usr/share/acrn/samples/nuc/launch_win.sh ./launch_debian.sh
+      $ sed -i "s/win10-ltsc.img/debian10.img/" launch_debian.sh
+      $ cp /usr/share/acrn/bios/OVMF.fd ./
+
+#. Assign USB ports to the Debian VM in order to use the mouse and keyboard before the launch:
+
+   .. code-block:: none
+
+      $ vim launch_debian.sh
+
+      <Add below as the acrn-dm parameter>
+      -s 7,xhci,1-2:1-3:1-4:1-5 \
+
+   .. note:: This will assign all USB ports (2 front and 2 rear) to the User VM. If you want to only assign the USB ports at the front, use "-s 7,xhci,1-2:1-3 \" instead. Refer to :ref:`acrn-dm_parameters` for ACRN for more information.
+
+#. Modify acrn.conf and reboot the Service VM to assign the Pipe A monitor to the Debian VM and the Pipe B monitor to the Service VM:
+
+   .. code-block:: none
+
+      $ sudo mount /dev/sda1 /mnt
+      $ sudo sed -i "s/0x01010F/0x010101/" /mnt/loader/entries/acrn.conf
+      $ sudo sed -i "s/0x011111110000/0x011100001111/" /mnt/loader/entries/acrn.conf
+      $ sed -i 3"s/$/ i915.enable_conformance_check=0/" /mnt/loader/entries/acrn.conf
+      $ sudo sync && sudo umount /mnt && reboot
+
+#. Copy grubx64.efi to bootx64.efi:

```

Comment:
@lirui34 , I forgot to ask earlier but why is this operation needed (copying `grubx64.efi` to `bootx64.efi`). Is OVMF.fd not able to use Grub directly?

Reply:
Yes, Debian VM not able to launch with the OVMF.fd unless we rename the `grubx64.efi` to `bootx64.efi`.

Reply:
Interesting, do you know why that's only happening with Debian but not with Ubuntu?

Reply:
I captured original contents of `ubuntu.img` and `debian.img`, for the debian vm, it's missing boot folder by default:
```
/mnt/EFI/
└── debian
    ├── BOOTX64.CSV
    ├── fbx64.efi
    ├── grub.cfg
    ├── grubx64.efi
    ├── mmx64.efi
    └── shimx64.efi

/mnt/EFI/
├── BOOT
│   ├── BOOTX64.EFI
│   └── fbx64.efi
└── ubuntu
    ├── BOOTX64.CSV
    ├── fw
    ├── fwupx64.efi
    ├── grub.cfg
    ├── grubx64.efi
    ├── mmx64.efi
    └── shimx64.efi
```